docs/config/index.rst

```diff
@@ -205,20 +225,22 @@ To see what configuration parameters are defined for a given ``conf``::
 
 You can also iterate through ``conf`` in a dictionary-like fashion::
 
-    >>> [key for key in conf.keys()]
+    >>> list(conf.keys())
```

Comment:
As in Python `dict`, `.keys()` is redundant, as shown above.
```suggestion
    >>> list(conf)
```

Reply:
This entire line is redundant, so I'll delete it.